src/waveresponse/_core.py

```diff
@@ -8,6 +8,18 @@
 from scipy.special import gamma
 
 
+def _robust_modulus(x, periodicity):
+    """
+    Robust modulus operator.
+
+    Similar to ``x % periodicity``, but ensures that it is robust w.r.t. floating
+    point numbers.
+    """
+    x = np.asarray_chkfinite(x % periodicity).copy()
+    np.nextafter(x, -1, where=(x == periodicity), out=x)
+    return x
```

Comment:
```suggestion
    return np.nextafter(x, -1, where=(x == periodicity), out=x)
```

Have no other impact then style. Thought it was a bit wierd to have a numpy function dingling there alone.